src/makeCalculator.js

```diff
@@ -5,6 +5,47 @@
  */
 function makeCalculator() {
```

Comment:
The function name `makeCalculator` is appropriate as it clearly describes the action of creating a calculator object.